### PR TITLE
Update Deck Stats Shown on Main Screen

### PR DESCRIPTION
**Pull request by wclausen**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java [hunk 1]
```diff
@@ -198,13 +206,68 @@ public void onBindViewHolder(ViewHolder holder, int position) {
...
         }
 
+        // Init vars to determine what metadata to show about decks
+        boolean showRemainingCards;
+        boolean showMinimumInterval;
+        try {
+            showRemainingCards = mCol.getConf().getBoolean("dueCounts");
+            // why isn't what follows another call to mCol.getConf()...?
+            // didn't want to modify anki collection prefs json object at this point
+            // so it must go through Android's preference system, which requires a Context object
```
Comment (z1lc) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java:216]: IMO the right call. Don't want to be modifying the general `conf` json

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java [hunk 2]
```diff
@@ -198,13 +206,68 @@ public void onBindViewHolder(ViewHolder holder, int position) {
...
+            holder.deckNew.setVisibility(View.INVISIBLE);
+            holder.deckLearn.setVisibility(View.INVISIBLE);
+            holder.deckRev.setVisibility(View.INVISIBLE);
+        }
+
+        // Show/hide the minimum interval information
+        if (showMinimumInterval) {
+            holder.minIvlLayout.setVisibility(View.VISIBLE);
+            holder.minIvl.setVisibility(View.VISIBLE);
+            int doneReviewingIvl = 14; // if minIvl is 14 days, you are g2g
```
Comment (z1lc) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: Can we change this to 21, to match [Anki's definition](https://apps.ankiweb.net/docs/manual.html#types-of-cards) of 'mature'? I know that's an arbitrary # too but I would rather standardize.

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java [hunk 3]
```diff
@@ -198,13 +206,68 @@ public void onBindViewHolder(ViewHolder holder, int position) {
...
+            } else {
+                holder.minIvl.setText(String.format("%dd", node.minIvl));
+            }
+            // also show any cards in learning stage
+            if (node.lrnCount > 0) {
+                holder.countsLayout.setVisibility(View.VISIBLE);
+                holder.deckLearn.setVisibility(View.VISIBLE);
+                holder.deckLearn.setText(String.format("(%d)", node.lrnCount));
+                holder.deckLearn
+                    .setTextColor((node.lrnCount == 0) ? mZeroCountColor : mLearnCountColor);
```
Comment (z1lc) [inline AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: General question: do we like the color red for these? To me red is associated with something bad, which a failed review really isn't (don't want to promote seeing failures as bad). Thoughts on going with another color?

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 4]
```diff
@@ -61,6 +61,9 @@
     <string name="button_size">Answer button size</string>
     <string name="card_browser_font_size">Card browser font scaling</string>
     <string name="col_path">AnkiDroid directory</string>
+    <string name="pref_show_min_interval" translatable="false">showMinInterval</string>
+    <string name="show_min_ivl_title" translatable="false">Show minimum interval</string>
+    <string name="show_min_ivl_summary" translatable="false">Shows/hides smallest interval for cards in deck</string>
```
Comment (z1lc) [inline AnkiDroid/src/main/res/values/10-preferences.xml:66]: nice touch

**Review by z1lc — APPROVED** [review]
looks great, amazing that you were able to get this out so quickly

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java (hunk 2 above)
Comment (wclausen) [reply AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: done

File: AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java (hunk 3 above)
Comment (wclausen) [reply AnkiDroid/src/main/java/com/ichi2/anki/widgets/DeckAdapter.java]: agree, red = bad. ultimately i think the absence of the 👍should communicate that you need to do work, so everything else can just be black?

fixed

**Comment by wclausen** [comment]
Also, important to note that the setting for "show remaining cards" in the "Reviewing" settings is what controls whether the card counts are shown on the deck list screen. It's not necessarily the cleanest approach, but it works. If we want to change that, lmk.

**Comment by z1lc** [comment]
Yeah not 100% intuitive but we can change that later if we want to iterate on this further. For now I think this is great. feel free to merge

**Pull request merged**